server/README.md

```diff
@@ -24,25 +24,37 @@ Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
 ```
 Make sure that `+json` is in the list of compiled features.
 
+### Natural Language Answers
+To execute natural language queries `bleep` needs to connect to a instance of [Qdrant](https://github.com/qdrant/qdrant), a vector search database. You can start Qdrant on port 6334 (where `bleep` expects it by default) with:
+
+```
+docker run -p 6333:6333 -p 6334:6334 \
+    -e QDRANT__SERVICE__GRPC_PORT="6334" \
+    qdrant/qdrant
+```
+
+You can run `bleep` without linking it to a `Qdrant` instance but calls to the `/answer` endpoint will return an error.
+
+You'll also need to run a local instance of `answer_api` (`bleep` expects it on port 7879 by default) which handles requests to the OpenAI API.
```

Comment:
can we link to the answer_api repo here?

Reply:
We need to open source first. Will do that and update after merging.